src/plugins/voiceMessages/index.tsx

```diff
@@ -71,6 +73,8 @@ const EMPTY_META: AudioMetadata = {
 
 function sendAudio(blob: Blob, meta: AudioMetadata) {
     const channelId = SelectedChannelStore.getChannelId();
+    const reply = PendingReplyStore.getPendingReply(channelId);
+    (reply && FluxDispatcher.dispatch({ type: "DELETE_PENDING_REPLY", channelId }));
```

Comment:
```suggestion
    if (reply) FluxDispatcher.dispatch({ type: "DELETE_PENDING_REPLY", channelId });
```
just a little more readable

Reply:
why not
```suggestion
    if (reply)
       FluxDispatcher.dispatch({ type: "DELETE_PENDING_REPLY", channelId });
```